pocket_ic/pocket_ic_server.py

```diff
@@ -109,44 +109,38 @@ def set_blob_store_entry(self, blob: bytes, compression: Optional[str]) -> str:
         """
         url = f"{self.url}/blobstore"
         if compression is None:
-            response = self.request_client.post(url, data=blob, headers=HEADERS)
+            response = self.request_client.post(url, data=blob)
         elif compression == "gzip":
-            headers = HEADERS | {"Content-Encoding": "gzip"}
+            headers = {"Content-Encoding": "gzip"}
             response = self.request_client.post(url, data=blob, headers=headers)
         else:
             raise ValueError('only "gzip" compression is supported')
 
         self._check_status_code(response)
         return response.text
 
-    def _get_url(self, pid: int) -> str:
-        tmp_dir = gettempdir()
-        ready_file_path = f"{tmp_dir}/pocket_ic_{pid}.ready"
-        port_file_path = f"{tmp_dir}/pocket_ic_{pid}.port"
-
+    def _get_url(self, port_file_path: int) -> str:
         stop_at = time.time() + 10  # Wait for the ready file for 10 seconds
```

Comment:
Should we time out here? The Rust library has no such timeout anymore.

Reply:
> I think a timeout makes sense, if the server did not come up within 10 seconds something is definitely wrong and we can terminate and inform the user, not?

This could lead to flakiness (which is why we dropped the timeout in the Rust library) on overloaded CI runners.